clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4192,6 +4193,142 @@ static bool ContainsWrapperAction(const Action *A) {
   return false;
 }
 
+// Put together an external compiler compilation call which is used instead
+// of the clang invocation for the host compile of an offload compilation.
+void Clang::ConstructHostCompilerJob(Compilation &C, const JobAction &JA,
```

Comment:
please add a comment about option processing strategy in presence of external host compiler. In
_clang++ -fsycl <`ClangOpts`>-fsycl-host-compiler=... -fsycl-host-compiler-options=<`ExtOpts`>_
command line, which of the `ClangOpts` are translated/passed to the host compiler binary (if any), are there restrictions/expectations what options `ClangOpts` and `ExtOpts` can contain respectively. E.g. what if both contain -o with conflicting name?